source/blog/2018-08-24-the-ember-times-issue-61.md

```diff
@@ -43,13 +43,13 @@ This new feature comes with two feature flags `store.shouldTrackAsyncRequests `
 
 ## [Podcast: Melanie Sumner on empowering JavaScript engineers 💪](https://twitter.com/samselikoff/status/1032298098901639169)
 
-This week [Sam Selikoff](https://github.com/samselikoff) interviewed Ember Core team member Melanie Sumner ([@MelSumner](https://github.com/MelSumner)) for [The EmberMap Podcast](https://embermap.com/). Melanie talked about her path from getting starting making Ember accessible to now being a leader in the Ember community as a core team member. Ember, for Melanie, was one of the first time in tech where she felt welcome. This is a huge part because of the welcoming Ember community.
+This week [Sam Selikoff](https://github.com/samselikoff) interviewed Ember Core team member Melanie Sumner ([@MelSumner](https://github.com/MelSumner)) for [The EmberMap Podcast](https://embermap.com/). Melanie talked about her path from getting starting making Ember accessible to now being a leader in the Ember community as a core team member. Ember, for Melanie, was one of the first times in tech where she felt welcome. This is a huge part because of the welcoming Ember community.
```

Comment:
"This is in huge part because of the welcoming Ember community"

Reply:
Good catch @locks! Maybe this sounds better?
"This is largely because of the welcoming Ember community"?

Reply:
🤦‍♂️ sorry about that – thanks for catching!

Reply:
Updated :-)